Services/DeviceModels.cs

```diff
@@ -78,6 +79,89 @@ public DeviceModel Get(string id)
             throw new ResourceNotFoundException();
         }
 
+        public DeviceModel OverrideDeviceModel(DeviceModel source, Models.Simulation.DeviceModelOverride overrideInfo)
```

Comment:
do we have a github issue tracking revisiting the design of the override classes?  The more I think of it the more I think they're going to get unwieldy & hard to maintain in the future.

Reply:
https://github.com/Azure/device-simulation-dotnet/issues/144